presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -171,6 +173,9 @@ public String getCaseSensitiveSchemaName(String caseInsensitiveSchemaName)
         for (TableMetadata table : keyspace.getTables()) {
             builder.add(table.getName());
         }
+        for (MaterializedViewMetadata materializedView : keyspace.getMaterializedViews()) {
```

Comment:
keyspace.getMaterializedViews() on pre-3.0 returns empty object without error like this.
<img width="858" alt="screen shot 2017-08-11 at 14 45 27" src="https://user-images.githubusercontent.com/6237050/29219110-0237ee3c-7ef2-11e7-93ea-71eddcaa410e.png">